hmpps-prisoner-search-indexer/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/indexer/services/dto/nomis/OffenderBooking.kt

```diff
@@ -40,8 +39,16 @@ data class OffenderBooking(
   val recall: Boolean? = null,
   val imprisonmentStatus: String? = null,
   val imprisonmentStatusDescription: String? = null,
-  val personalCareNeeds: List<PersonalCareNeed>? = null,
   val receptionDate: LocalDate? = null,
   val locationDescription: String? = null,
   val latestLocationId: String? = null,
-)
+) : Diffable<OffenderBooking> {
+  override fun diff(other: OffenderBooking): DiffResult<OffenderBooking> = getDiffResult(this, other)
+}
+
+private fun getDiffResult(booking: OffenderBooking, other: OffenderBooking): DiffResult<OffenderBooking> =
+  DiffBuilder(booking, other, ToStringStyle.JSON_STYLE).apply {
+    OffenderBooking::class.members
+      .filterNot { listOf("diff", "equals", "toString", "hashCode").contains(it.name) }
+      .forEach { property -> append(property.name, property.call(booking), property.call(other)) }
+  }.build()
```

Comment:
Made the new model diffable so we can compare them - but not done anything with it yet, coming in next PR